src/hooks/doctor.hook.tsx

```diff
@@ -0,0 +1,18 @@
+import {useState, useEffect} from "react"
+import {Doctor} from "../models/doctor"
+import {doctorsList} from "../helpers/doctorsList"
+
+const useDoctors = () => {
+    const [doctors, setDoctors] = useState<Doctor[]>([]);
+    const [spinner, setSpinner] = useState<boolean>(true);
```

Comment:
`loading` might be a better naming here
```suggestion
    const [loading, setLoading] = useState<boolean>(true);
```